MCprep_addon/world_tools.py

```diff
@@ -150,7 +150,7 @@ class WorldExporter(Enum):
 }
 
 
-def get_exporter(context: Context) -> Optional[WorldExporter]:
+def get_exporter(context: Context) -> WorldExporter:
```

Comment:
Like I mentioned earlier, the `Optional` typing allows for more nuance.

`WorldExporter.Unknown` should mean that an exporter is detected, but not one that has extensive testing with MCprep (like a new exporter). `None` should mean no exporter has been detected at all. 

Thus, I'd rather we keep the Optional typing

Reply:
Hm I can see that argument. Though it means multiple checks against None _and_ Unknown I guess we could define the "supported" enums in a global somewhere and check if the response is in that?

Reply:
We could also create a "unsupported" global with the following:
```py
UNSUPPORTED_OR_NONE = (WorldExporter.Unknown, None)
```

And check against that for checks that require blocking both. The reason I'd rather keep `Optional` though is because there are certain operations that don't necessarily require an exporter to be supported, but do need it to be detected, such as textureswap. For those operations, collapsing an unknown exporter and none being detected to a single enum would make things a little more convoluted.

Reply:
Ok I like that solution. I'll make a commit for this soon

Reply:
Welp we both came up with the same commit 😅 using yours since it's already on remote, resolving this thread